### DISCUSSION TITLE
Support hebrew in Nova model?

### DISCUSSION THREAD

**Discussion by aviadr1** [deepgram/community#903] [Feature Requests]
Thanks for an awesome product!

Would it be possible to add hebrew as a supported language for Nova model? We can supply a fully free extensive dataset for training models in hebrew transcription. it provides a trained whisper model and also the dataset to train it:

The non profit ivrit.ai org is providing high quality dataset with completely free license that is also applicable to commerical use. 
dataset & trained whisper model: https://huggingface.co/ivrit-ai
license: https://www.ivrit.ai/en/the-license/

the Ivrit team is happy to provide any other resources to make hebrew well supported both in opensource and commercial products.
My company Gigaverse would like to use Nova model with hebrew transcription support in our product alongside the other 36 languages that Nova supports.

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by eltoob** [comment]
Would love also hebrew support!

**Reply by eltoob** [reply]
By the way, an option would be to allow to stream whisper as we found whisper medium actually works pretty well with hebrew (but not as well as ivrit-ai)

**Comment by jpvajda** [comment]
Closing as this is a duplicate of: https://github.com/orgs/deepgram/discussions/219
